tests/test_channel_metrics/test_completely_bounded_trace_norm.py

```diff
@@ -9,27 +10,31 @@
 def test_cb_trace_norm_quantum_channel():
     """The diamond norm of a quantum channel is 1."""
     phi = dephasing(2)
-    np.testing.assert_equal(completely_bounded_trace_norm(phi), 1)
+    assert completely_bounded_trace_norm(phi) == 1
+
+
+def test_cb_trace_norm_CP():
+    """Test for the diamond norm of a CP map."""
+    non_normalized_depolarizing_array = np.eye(4)
+    assert completely_bounded_trace_norm(non_normalized_depolarizing_array) == 4.0
 
```

Comment:
I did the lazy thing and used an un-normalized quantum channel. Let me know if I should think of other `np.array` examples.

Reply:
Honestly, I think the "lazy" thing here is the right thing to do. Nice!